File: submissions/sm_022_prakash/week_03/day_3/submissions_2/split_string.js
givenString = 'masai school is better'
var outArray = []
splitchar = 'a'
var str = "";
//function to split string at the given character
function splitString(givenString){
for (var i = 0; i < givenString.length; i++) {

    if (givenString[i] != splitchar) {

        str += givenString[i];
    }
    else {
        outArray.push(str);
        str = "";
    }
}
outArray.push(str);
console.log(outArray)
return splitString;
}


splitString(givenString)

File: submissions/sm_012_gaurav/week_13/day_4/session_1/split_string.py
inputString = input("Enter String: ")
splitChar = input("Enter Split Character: ")
l = []
s = ''
for a in inputString:
    if a != splitChar:
        s += a
    else:
        if len(s) != 0:
            l.append(s)
            s = ''
l.append(s)
print(l)